experimental/PropertyDDS/packages/property-dds/src/propertyTreeExtFactories.ts

```diff
@@ -80,25 +159,57 @@ export class DeflatedPropertyTreeFactory implements IChannelFactory {
         return DeflatedPropertyTreeFactory.Attributes;
     }
 
+    public getEncodeFce() { return deflate; }
+    public getDecodeFce() { return inflate; }
+    public newPropertyTree(
+        id: string,
+        runtime: IFluidDataStoreRuntime,
+        attributes: IChannelAttributes,
+        options: SharedPropertyTreeOptions,
+        propertyTreeConfig: IPropertyTreeConfig): SharedPropertyTree {
+        return new DeflatedPropertyTree(id, runtime, attributes, options, propertyTreeConfig);
+    }
+}
+
+export class LZ4PropertyTreeFactory extends CompressedPropertyTreeFactory {
+    public static readonly Type = "LZ4PropertyTree:84534a0fe613522101f6";
+
+    public static readonly Attributes: IChannelAttributes = {
+        type: LZ4PropertyTreeFactory.Type,
+        snapshotFormatVersion: "0.1",
+        packageVersion: "0.0.1",
+    };
+
     public async load(
         runtime: IFluidDataStoreRuntime,
         id: string,
         services: IChannelServices,
         attributes: IChannelAttributes,
         url?: string,
-    ): Promise<DeflatedPropertyTree> {
-        const options = {};
-        const instance = new DeflatedPropertyTree(id, runtime, attributes, options as SharedPropertyTreeOptions
-            , { encDec });
-        await instance.load(services);
-        return instance;
+    ): Promise<LZ4PropertyTree> {
+        return await super.load(runtime, id, services, attributes, url) as DeflatedPropertyTree;
     }
 
-    public create(document: IFluidDataStoreRuntime, id: string, requestUrl?: string): DeflatedPropertyTree {
-        const options = {};
-        const cell = new DeflatedPropertyTree(id, document,
-            this.attributes, options as SharedPropertyTreeOptions, { encDec });
-        cell.initializeLocal();
-        return cell;
+    public create(document: IFluidDataStoreRuntime, id: string, requestUrl?: string): LZ4PropertyTree {
+        return super.create(document, id, requestUrl);
+    }
+
+    public get type() {
+        return DeflatedPropertyTreeFactory.Type;
+    }
```

Comment:
Should be LZ4PropertyTree.Type and .Attributes

Reply:
@DLehenbauer thank you for finding this